### PR TITLE
Adding an argument to choose which answer will skip to the next song + fix scoreboard

### PR DESCRIPTION
**Pull request by peyrlac**
I think it could be useful for Disney music quiz because we don't know the singer or to add more difficulty.
It's still able to count points if they find the artist and the title. It'll be extra points (to be change if you don't like)
I fixed also the scoreboard for more than 3 players.

### CODE REVIEW

**Comment by nicoeg** [comment]
Thanks for this. I'm currently on vaccation with shitty internet. Will have a look once I get home

File: src/music-quiz.ts [hunk 1]
```diff
@@ -206,6 +214,8 @@ export class MusicQuiz {
                     position = ':second_place:'
                 } else if (index === 3) {
                     position = ':third_place:'
+                } else if (index > 3) {
```
Comment (nicoeg) [inline src/music-quiz.ts]: What's the issue with the scoreboard - can you show me an image of what it prints? 

It should show positions above 3 with bold: `let position = `**${index + 1}.** ``

Comment (peyrlac) [reply src/music-quiz.ts]: It shows this https://ibb.co/1nf6x62
For the other suggestion, it's possible that the code doesn't work. I'm not familiar with the language but it's an idea that can be add.

File: src/music-quiz.ts [hunk 2]
```diff
@@ -206,6 +214,8 @@ export class MusicQuiz {
...
+                    position = index.'.'
```
Comment (nicoeg) [inline src/music-quiz.ts]: Oh I see... The 3 should just be a two :)

```suggestion
                } else if (index === 2) {
                    position = ':third_place:'
```

File: src/commands.ts [hunk 3]
```diff
@@ -23,6 +23,12 @@ export class MusicQuizCommand extends Command {
                     prompt: 'How many songs the quiz will contain',
                     type: 'string',
                     default: 10
+                },
+                {
+                    key: 'onlyThis',
+                    prompt: 'Only this answer is required; artist, title or both',
+                    type: 'string',
+                    default: 'both'
```
Comment (nicoeg) [inline src/commands.ts]:
```suggestion
                    key: 'only',
                    prompt: 'Only this answer is required; artist, title or both',
                    type: 'string',
                    oneOf: ['artist', 'title', 'both'],
                    default: 'both'
```

**Review by nicoeg — CHANGES_REQUESTED** [review]
Just a couple of changes - will merge once adressed

**Comment by nicoeg** [comment]
Thanks again - will make some minor adjustments and then deploy.

I'll let you know once deployed

**Comment by nicoeg** [comment]
@MrLacpierre Update is deployed now

**Pull request merged**